compiler/modules/front/codeGen.bal

```diff
@@ -1172,7 +1172,7 @@ function codeGenCallStmt(CodeGenContext cx, bir:BasicBlock startBlock, Environme
         return check codeGenCheckingStmt(cx, startBlock, env, expr.checkingKeyword, expr.operand, expr.kwPos);
     }
     if result != () {
-        return cx.semanticErr("return type of function or method in call statement must be nil", stmt.startPos);
+        return cx.semanticErr("return type of function or method in call statement must be nil", s:range(stmt));
```

Comment:
Just do expressions to start with as I suggested: https://github.com/ballerina-platform/nballerina/issues/763#issuecomment-1003478084

Reply:
fixed